third_party/xnnpack.buck.bzl

```diff
@@ -4,6 +4,7 @@ load("//tools/build_defs:glob_defs.bzl", "subdir_glob")
 load("//tools/build_defs:platform_defs.bzl", "ANDROID", "APPLE", "APPLETVOS", "CXX", "IOS", "MACOSX", "WINDOWS")
```

Comment:
we use the same buck definitions internally for xplat builds. Is this going to work internally if we do this?

Reply:
Yeah, I'm hoping to merge directly onto the release branch, which is separate from internal. If we can do that, it won't affect internal builds. I'll update internal again soon to resolve cleanly, but I have to deal with the larger build changes they made in the last few weeks.